.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,16 @@
+{
+    "version": 4,
+    "configurations": [
+        {
+            /** 
+             * ms-vscode.cmake-tools plugin should be installed.
+             * 
+             * It provides data for C/C++ plugin,
+             * such as include paths, browse paths, defines, etc.
+             */
+            "name": "STM32",
+            "configurationProvider": "ms-vscode.cmake-tools",
+            "intelliSenseMode": "${default}"
+        }
+    ]
+}
```

Comment:
@miDeb Will this change be a problem for your vscode setup?

Reply:
I don't think so